packages/lib/constants.ts

```diff
@@ -141,3 +144,5 @@ export const BOOKER_NUMBER_OF_DAYS_TO_LOAD = parseInt(
   process.env.NEXT_PUBLIC_BOOKER_NUMBER_OF_DAYS_TO_LOAD ?? "0",
   0
 );
+
+export const MINIMUM_NUMBER_OF_ORG_SEATS = 30;
```

Comment:
Abstracted the number out here.

Reply:
Nice @PeerRich were you wanting to remove this limit of org seats? Or just for admins

Reply:
It has to be there for self serve. Admins can choose a lower number